libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,6 +88,16 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
+	// The sequence number of the global Keybase Merkle tree at the
+	// time this update was created (from the writer's perspective).
+	// This field was added to V3 after it was live for a while, and
+	// older clients that don't know about this field yet might copy
+	// it into new updates via the unknown fields copier. Which means
+	// new MD updates might end up referring to older Merkle roots.
+	// That's ok since this is just a hint anyway, and shouldn't be
```

Comment:
Can non-legitimate writer with old client have old seq number included in the metadata, making their non-legitimate write seem legitimate? Or more importantly, can an attacker use a modified client to write to a TLF using an identity that is not a writer of the TLF any more? I just realized these two are the same case, so it doesn't really matter that much that we have old clients that simply copy the field without ever updating it.

Is the solution to these concerns (in the future when we implement it) what makes the verification more expensive?